### DISCUSSION TITLE
SSD Temperature Sensors missing...

### DISCUSSION THREAD

**Discussion by EyoSama** [Rem0o/FanControl.Releases#1556] [Q&A]
I know that FanCiontrol uses LibreHardwareMonitor, but I noticed that the sensors that show up when you run the LibreHardwareMonitor app, don't all show up in FanControl, important ones for example being the SSD temps. Is it possible that FanControl is using an old version of LHM that didn't support these other sensors, or is there some other issue?

**Comment by Rem0o** [comment]
FanControl uses the latest commit for LHM. Make sure you got all sensor types enabled in sensor settings.

**Reply by EyoSama** [reply]
After refreshing it, the sensors showed up!! Btw, amazing work on this incredible piece of software! I have never been happier to donate to a great project! Thanks.
